commit 46e2d4d1bd2c17e2f84dd90768321ee0bbaa6b8a
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Sep 14 16:16:24 2017 +0100

    config: Add CONFIG_GPIO_WATCHDOG=m
    
    Enable the gpio_wdt module, to support external watchdogs.
    
    See: https://github.com/raspberrypi/linux/issues/2199
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 1cbfeb00889c..74293f19e7d3 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -668,6 +668,7 @@ CONFIG_SENSORS_TMP102=m
 CONFIG_THERMAL=y
 CONFIG_THERMAL_BCM2835=y
 CONFIG_WATCHDOG=y
+CONFIG_GPIO_WATCHDOG=m
 CONFIG_BCM2835_WDT=y
 CONFIG_UCB1400_CORE=m
 CONFIG_MFD_STMPE=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index d0a9d90b6873..6625fdb53f4e 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -662,6 +662,7 @@ CONFIG_SENSORS_TMP102=m
 CONFIG_THERMAL=y
 CONFIG_THERMAL_BCM2835=y
 CONFIG_WATCHDOG=y
+CONFIG_GPIO_WATCHDOG=m
 CONFIG_BCM2835_WDT=y
 CONFIG_UCB1400_CORE=m
 CONFIG_MFD_STMPE=y
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1689_46e2d4d1bd2c17e2f84dd90768321ee0bbaa6b8a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1689_46e2d4d1bd2c17e2f84dd90768321ee0bbaa6b8a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
